docs/development/extensions-core/s3.md

```diff
@@ -61,6 +63,7 @@ As an example, to set the region to 'us-east-1' through system properties:
 |`druid.storage.sse.type`|Server-side encryption type. Should be one of `s3`, `kms`, and `custom`. See the below [Server-side encryption section](#server-side-encryption) for more details.|None|
 |`druid.storage.sse.kms.keyId`|AWS KMS key ID. This is used only when `druid.storage.sse.type` is `kms` and can be empty to use the default key ID.|None|
 |`druid.storage.sse.custom.base64EncodedKey`|Base64-encoded key. Should be specified if `druid.storage.sse.type` is `custom`.|None|
+|`druid.storage.type`|Global deep storage provider. Must be set to `s3` to make use of this extension.|`local`| 
```

Comment:
The default value is `local` and so this doc is technically correct. But, I'm a bit worried about whether it could confuse people. How about changing it to `Must be set.` instead of `local`?

Also, I guess you added it here based on an alphabetical order but there's no actual ordering I think. How about moving it to the top of the table since it must be set?

Reply:
Yes that probably makes sense to just mark it as mandatory. We could suggest people might want to use `s3` considering the page they're viewing.
Regarding the table ordering it looks like the knobs are grouped at least by their namespace so I put it with the other `druid.storage` parameters. Seeing that the first paragraph in the Configuration section mentions that this knob is mandatory it seems to me one would be able to quickly grasp the importance of this parameter.

Reply:
Ok, it sounds good to me.